CSS/Drawing/css-pseudo-elements.md

```diff
@@ -0,0 +1,121 @@
+# CSS pseudo element
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More about pseudo elements](https://www.html5rocks.com/en/tutorials/shapes/getting-started/)'
+
+---
+## Content
+
+*Pseudo elements* are used in CSS to style specified parts of an element.
+
+By using `::first-letter` pseudo element we can style the first letter of a text. For example we can style the first letter of all `<p>` elements:
```

Comment:
The `::first-letter` pseudo element styles the first letter of any text element. For example, we can style the first letter of all `<p>` elements: